File: Payroll/Services/LoginService.cs
using System;
using Payroll.Models;

namespace Payroll.Services
{
    public class LoginService
    {
        private PayrollEntities db;

        public const String LoggedUser = "LoggedUser";

        public Boolean IsUserValid(String username, String userpass)
        {
            var securityService = new SecurityService();
            String hashedPassword = securityService.HashString(userpass);
            m_User userInDb;
            Boolean isUserValid = false;
            using (db = new PayrollEntities())
            {
                userInDb = db.m_User.Find(username);
                if (userInDb != null)
                    isUserValid = userInDb.Password == hashedPassword;
            }

            return isUserValid;
        }
    }
}

File: payroll-sveltekit/src/lib/services/loginService.ts
import prisma from "$lib/prisma";
import { hashString } from "./securityService";

export async function isUserValid (username: string, userpass: string) : Promise<boolean> {
    let hashedPwd: string = hashString(userpass);
    let m_User = await prisma.m_User.findMany({
        where: {
            Password: hashedPwd
        }
    });

    return (m_User.length > 0);
}

// export async function createSession (params:type) => {
    
// }

// export async function getUserFromSession (params:type) => {
    
// }